### ISSUE TITLE
update libuv when it incorporates statx fix

### ISSUE THREAD

**Issue by owaddell** [becls/swish#27]
**Platform**: Linux toolbox 5.0.7-200.fc29.x86_64 #1 SMP Mon Apr 8 15:40:59 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

Get [this libuv pull request](https://github.com/libuv/libuv/pull/2263) merged to fix:
```
$ valgrind ./build/release/bin/swish 
==2577== Memcheck, a memory error detector
==2577== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2577== Using Valgrind-3.14.0 and LibVEX; rerun with -h for copyright info
==2577== Command: ./build/release/bin/swish
==2577== 
==2577== Conditional jump or move depends on uninitialised value(s)
==2577==    at 0x4BF57D: Sunsigned64 (in /home/owaddell/projects/swish/build/release/bin/swish)
==2577==    by 0x41401C: stat_cb (osi.c:315)
==2577==    by 0x5282B0: uv__work_done (threadpool.c:313)
==2577==    by 0x52C386: uv__async_io (async.c:118)
==2577==    by 0x53994E: uv__io_poll (linux-core.c:379)
==2577==    by 0x52CB7D: uv_run (core.c:363)
==2577==    by 0x41709C: osi_get_callbacks (osi.c:1102)
==2577==    by 0x9328832: ???
==2577==    by 0x4CEE01F: ???
==2577==    by 0x4CCCA0: S_call_help (in /home/owaddell/projects/swish/build/release/bin/swish)
==2577==    by 0x4CE4DF: boot_call (in /home/owaddell/projects/swish/build/release/bin/swish)
==2577==    by 0x4CE6E5: load (in /home/owaddell/projects/swish/build/release/bin/swish)
==2577== 
```

**Comment by owaddell** [comment]
The libuv folks have merged my PR so we can close this.

**Comment by owaddell** [comment]
Oops, I got confused by the Merged icon beside my swish branch that showed up as referenced over on the libuv repository.

**Comment by burgerrg** [comment]
You may need to ask someone at libuv what the process is for getting your approved changes into the main line. I think I added a comment to my pull request when it got to this point, and that spawned the appropriate process.

**Comment by owaddell** [comment]
The libuv folks landed the fix in 1c2dc9c8d10aff2d63734fb5cb01eefcc19f6e1b. Looks like we'll pick it up in v1.28.1.

**Comment by burgerrg** [comment]
libuv v1.29.0 from May 16, 2019, has the fix in it.
